src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -217,6 +217,26 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         return versions[0] ? versions[0].id : null;
     };
 
+    mergeVersions = (newVersion: BoxItemVersion) => {
+        const { versions } = this.state;
+        const newVersions = cloneDeep(versions).map(version =>
+            version.id !== newVersion.id ? version : { ...version, ...newVersion },
```

Comment:
We usually want conditions to assert `is` before or instead of `is not` when possible. Can we switch this to `===`?